src/SelfService/Application/MembershipApplicationService.cs

```diff
@@ -97,14 +109,7 @@ public async Task<MembershipId> AcceptApplication(MembershipApplicationId applic
             throw EntityNotFoundException<Capability>.UsingId(membershipApplication.CapabilityId);
         }
 
-        var newMembership = Membership.CreateFor(
-            capabilityId: membershipApplication.CapabilityId,
-            userId: membershipApplication.Applicant,
-            createdAt: _systemTime.Now
-        );
-
         await CreateAndAddMembership(membershipApplication.CapabilityId, membershipApplication.Applicant);
```

Comment:
No longer need the id?

Reply:
This id is returned but never actually used by anything. (I am quite sure)

Besides that id is not the id we store, as `CreateAndAddMembership` also generates a new object.